README.adoc

```diff
@@ -198,20 +199,26 @@ include::{common-includes}/mvnpackage.adoc[]
 
 == Existing Tracer implementation
 
-In order to collect traces across your systems, you are required to implement the OpenTracing `Tracer`
+To collect traces across your systems, you are required to implement the OpenTracing `Tracer`
 interface. For the purposes of this guide, we are providing a bare-bones `Tracer` implementation for
 the Zipkin server in a form of a user feature for Open Liberty.
 
-This feature has already been configured for you and it will be installed automatically into each service
-when you build them. You can find it enabled in your `server.xml` files:
+This feature is already configured for you in your `pom.xml` and `server.xml` files. It will be
+downloaded and installed automatically into each service when you run a Maven build. You can find it
+enabled in your `server.xml` files as:
+
+// This feature is already configured for you and it will be installed automatically into each service
+// when you build them. You can find it enabled in your `server.xml` files:
 
 [source, xml, role="no_copy"]
 ----
 <feature>usr:opentracingZipkin-0.30</feature>
 ----
 
-The user feature is pulled in and installed automatically using the following Maven plugin in `system/pom.xml` and
-`inventory/pom.xml`:
+Your `pom.xml` files download and install it using the following Maven plugin:
```

Comment:
The "it" is throwing me off in this sentence.

"Your `pom.xml` files download and installed by using the following Maven plugin:"?
"Your `pom.xml` files download. Install them by using the following Maven plugin:"?
"Download and install your `pom.xml` files by using the following Maven plugin:"?

Reply:
I just got confused myself as well lol... it should say "Your pom.xml files download and install this user feature by using the following Maven plugin:"

Reply:
Actually, perhaps we can instead say "The Maven plugin responsible for download and installing the user feature is:"